### PR TITLE
DEVX-5442 Add forceMuteStream and forceMuteAll

### PR DESCRIPTION
**Pull request by moficodes**
What is this PR doing?
This PR serves as the implementation of forceMuteStream and forceMuteAll methods.

How should this be manually tested?
What are the relevant tickets?
https://jira.vonage.com/browse/DEVX-5442

### CODE REVIEW

**Review by pardel — DISMISSED** [review]

File: src/main/java/com/opentok/MuteAllProperties.java [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+import java.util.stream.Collectors;
+
+/**
+ * Defines values for the <code>properties</code> parameter of the
+ * {@link OpenTok#forceMuteAll(String, MuteAllProperties)} method.
+ *
+ * @see OpenTok#forceMuteAll(String, MuteAllProperties)
+ */
+public class MuteAllProperties {
+    private List<String> excludedStreams;
```
Comment (jeffswartz) [inline src/main/java/com/opentok/MuteAllProperties.java:21]: Maybe this should be called `excludedStreamIds`. It is a list of stream IDs (strings) not Stream objects. (The Java SDK does include a Stream class.)

File: src/main/java/com/opentok/MuteAllProperties.java [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+         * Call this method to add a list of Stream to the excludedStreams list
+         *
+         * @param streams A List of type {@link Stream}.
+         *
+         * @return The MuteAllProperties.Builder object with excludedStreams list.
+         */
+        public MuteAllProperties.Builder excludedStreams(List<Stream> streams) {
+            this.excludedStreams.addAll(streams.stream().map(stream -> stream.getId()).collect(Collectors.toList()));
+            return this;
+        }
```
Comment (jeffswartz) [inline src/main/java/com/opentok/MuteAllProperties.java:57]: I don't think we need this method. I think just including the `MuteAllProperties.Builder excludedStreamIds(List<String> ids)` method suffices.

File: src/main/java/com/opentok/MuteAllProperties.java [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+         * Call this method to add a stream id to the excludedStreams list
+         *
+         * @param id An object of type {@link String}.
+         *
+         * @return The MuteAllProperties.Builder object with excludedStreams list.
+         */
+        public MuteAllProperties.Builder excludedStreamId(String id) {
+            this.excludedStreams.add(id);
+            return this;
+        }
```
Comment (jeffswartz) [inline src/main/java/com/opentok/MuteAllProperties.java:69]: I don't think we need this method. I think just including the `MuteAllProperties.Builder excludedStreamIds(List<String> ids)` method suffices.

File: src/main/java/com/opentok/MuteAllProperties.java [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+    }
+
+    /**
+     * Returns the excludedStreams List
+     *
+     * @return the excludedStreams list
+     */
+    public List<String> getExcludedStreams() {
+        return this.excludedStreams;
+    }
```
Comment (jeffswartz) [inline src/main/java/com/opentok/MuteAllProperties.java:88]: I think this should be `getExcludedStreamIds()`. It is a list of stream IDs (strings), not Stream objects.

**Review by jeffswartz — CHANGES_REQUESTED** [review]

File: src/main/java/com/opentok/MuteAllProperties.java [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+     * @see MuteAllProperties
+     */
+    public static class Builder {
+        private List<String> excludedStreamIds = new ArrayList<>();
+
+        /**
+         * Call this method to add a A List of stream IDs for streams to be excluded
+         * from the force mute action.
+         *
+         * @param ids The List of stream IDs,
```
Comment (jeffswartz) [inline src/main/java/com/opentok/MuteAllProperties.java:39]:
```suggestion
         * @param ids The List of stream IDs.
```

File: src/main/java/com/opentok/OpenTok.java [hunk 6]
```diff
@@ -630,19 +631,68 @@ public void setStreamLayouts(String sessionId, StreamListProperties properties)
...
+    /**
+     * Force publishers of all streams in a session to mute audio. You can exclude a list of
+     * streams from being muted using the MuteAllProperties class.
+     *
+     * <p>
+     * For more information, see
+     * <a href="https://tokbox.com/developer/guides/moderation/#force_mute">Muting the audio of streams in a session</a>.
+     *
+     * @param sessionId The session ID.
+     * @param properties Defined a list of stream IDs for streams that should be excluded
```
Comment (jeffswartz) [inline src/main/java/com/opentok/OpenTok.java:679]:
```suggestion
     * @param properties Defines a list of stream IDs for streams that should be excluded
```

File: src/main/java/com/opentok/MuteAllProperties.java [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+/**
+ * Defines values for the <code>properties</code> parameter of the
+ * {@link OpenTok#forceMuteAll(String, MuteAllProperties)} method.
+ *
+ * @see OpenTok#forceMuteAll(String, MuteAllProperties)
+ */
+public class MuteAllProperties {
+    private List<String> excludedStreamIds;
+
+    private MuteAllProperties(MuteAllProperties.Builder builder) {
```
Comment (jeffswartz) [inline src/main/java/com/opentok/MuteAllProperties.java:23]: We also need methods related to the `active` option for this method. See the `active` option for the REST method: https://tokbox.com/developer/rest/#force_mute_session.

**Review by jeffswartz — CHANGES_REQUESTED** [review]

File: src/main/java/com/opentok/MuteAllProperties.java (hunk 7 above)
Comment (moficodes) [reply src/main/java/com/opentok/MuteAllProperties.java:24]: added active option and tests.

**Review by jeffswartz — APPROVED** [review]

**Pull request merged**